Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Material/MaterialAsset.h

```diff
@@ -121,11 +126,20 @@ namespace AZ
             // MaterialReloadNotificationBus overrides...
             void OnMaterialTypeAssetReinitialized(const Data::Asset<MaterialTypeAsset>& materialTypeAsset) override;
 
+            static const char* s_debugTraceName;
```

Comment:
Should this be available only in debug/profile build?  is it still required or only for testing?